src/main/webapp/app/shared/dashboards/tutor-leaderboard/tutor-leaderboard.component.ts

```diff
@@ -38,6 +38,9 @@ export class TutorLeaderboardComponent implements OnInit {
         if (this.exercise && this.exercise.course) {
             this.course = this.exercise.course;
             this.isExerciseDashboard = true;
+        } else if (this.exercise && this.exam && this.exam.course) {
+            this.course = this.exam.course;
+            this.isExerciseDashboard = true;
         }
         this.isExamMode = !!this.exam;
```

Comment:
You can simplify that pice by quite a bit:
```suggestion
        this.course = getCourseFromExercise(exercise);
        this.isExerciseDashboard = !!(this.exercise && this.course);
        this.isExamMode = !!this.exam;
```